pandas/core/config_init.py

```diff
@@ -371,6 +378,8 @@ def table_schema_cb(key):
                        validator=is_int)
     cf.register_option('html.use_mathjax', True, pc_html_use_mathjax_doc,
                        validator=is_bool)
+    cf.register_option('html.urls_as_links', False, pc_html_urls_as_links_doc,
```

Comment:
I don't think an option is suitable here; better just served as a keyword argument

Reply:
Agreed.  I might even go as far as to consider reducing the number of available `html.*` options and leave them to be configured in the signature.

Reply:
Thanks for the review. I should have mentioned I worked off an [older pull request](https://github.com/pandas-dev/pandas/pull/9880), where this was requested. I'm assuming it's so notebook users can enable the option and have links render when a cell containing just a DF name is executed.

Reply:
Makes sense, but over time we've slightly moved away from that type of global option management. Keyword route should be much simpler from a end user and maintenance perspective here

Reply:
Ah ok. In that case, I don't see how a notebook user would enable this behavior in a notebook (since the method call happens behind the scenes). If the use case isn't supported, I'll remove that part of the test.